Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -354,7 +356,10 @@ def audit_license
         return if user.blank?
 
         github_license = GitHub.get_repo_license(user, repo)
-        return if github_license && (formula.license + ["NOASSERTION"]).include?(github_license)
+        if github_license.blank? || github_license == "NOASSERTION" ||
+           formula.license.any? { |license| license.start_with? github_license }
+          return
+        end
```

Comment:
As far as I know, `github_license` can be one of these three options:

1. `nil` if the repo has not specified any license (e.g. [`ptex`](https://github.com/wdas/ptex))
    - This is okay because there still may be a license that isn't specified
1. `"NOASSERTION"` if the repo has a license file but GitHub can't determine what license it is (e.g. [`pacapt`](https://github.com/icy/pacapt))
1. An actual license like `"GPL-3.0"` (e.g. [`cask`](https://github.com/cask/cask))

We only want to compare the licenses the third situation.

Reply:
This seems reasonable 👍🏻